### ISSUE TITLE
CIS Scan - Choose a profile dropdown is empty when trying to run as Cluster Owner

### ISSUE THREAD

**Issue by izaac** [rancher/rancher#25990]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**

1. Create an RKE cluster
2. Create an **Standard User** in Rancher
3. Assign the user from Step 2 as the cluster owner of the cluster of Step 1
4. Logout from Admin and Login as the Cluster Owner added in Step 3
5. In the Cluster navigate to Run CIS Scan

**Result:**

The profile dropdown will be empty. As admin I can see it populated.

<img width="607" alt="Screen Shot 2020-03-12 at 1 46 47 PM" src="https://user-images.githubusercontent.com/27659/76566333-c8b6fa80-6469-11ea-84e3-036057d10f75.png">

**Additional Information:**

**/v3/cisconfigs** is empty for this Standard User.

As an admin this is the respond coming from /v3/cisconfigs:

```json
CisConfigParams": {
  "default": {
   "benchmarkVersion": "rke-cis-1.4"
  },
  "v1.15": {
   "benchmarkVersion": "rke-cis-1.4"
  },
  "v1.16": {
   "benchmarkVersion": "rke-cis-1.4"
  },
  "v1.17": {
   "benchmarkVersion": "rke-cis-1.4"
  },
  "v1.18": {
   "benchmarkVersion": "rke-cis-1.4"
  }
```

**Environment information**
- Rancher version: `2.4 master-head (03/12/2020)` commit id: d47b0ec4c
- Installation option (single install/HA): single

**Comment by izaac** [comment]
Rancher version: `v2.4.0-rc8 (03/20/2020)`

I created `admin1` as admin user in rancher.
Login with admin1 and created a cluster.
Added `user1` and `user2` standard Rancher users.
Added user1 as Cluster Member to the cluster.
Added user2 as Cluster Owner to the cluster.

![Screen Shot 2020-03-20 at 10 20 10 AM](https://user-images.githubusercontent.com/27659/77189439-cae60e00-6a94-11ea-963b-9089ba7193e6.png)


user1 (Cluster Member) can't see the Run CIS scan option in dropdown or click in Run in Tools -> CIS Scans.
user2 (Cluster Owner) can click the Run CIS scan but the CIS Profile dropdown is empty. And neither from Tools -> CIS Scan

![Screen Shot 2020-03-20 at 10 20 26 AM](https://user-images.githubusercontent.com/27659/77189596-0da7e600-6a95-11ea-97a3-9c3c1ad2188d.png)

**Comment by sangeethah** [comment]
Need to wait for rancher server build that has the fix.

**Comment by izaac** [comment]
Rancher version: `2.4 master-head (03/20/2020)` commit id: dea9bf3bf

This is now passing.

1. I created admin1 as admin user in rancher.
2. Login with admin1 and created a cluster.
3. Added user1 and user2 standard Rancher users.
4. Added user1 as Cluster Member to the cluster.
5. Added user2 as Cluster Owner to the cluster.

![Screen Shot 2020-03-20 at 10 20 10 AM](https://user-images.githubusercontent.com/27659/77193731-eef91d80-6a9b-11ea-8c97-1ac8ee1ce2a6.png)

user2 (Cluster Owner) is now able to select a profile and run the scans listed in the CIS Profile dropdown. Admin user is also able to list and run scans.

user1 (Cluster Member) is not able to run scans from Tools -> CIS Scans, Global view and Cluster Dashboard. (Expected).

Tried from Global view, Cluster Dashboard and Tools -> CIS Scans.
